Cargo.toml

```diff
@@ -26,12 +26,13 @@ flate2 = { version = "1.0.3", default-features = false, features = ["zlib"] }
 home = "0.5.3"
 libc = "0.2"
 nom = "7.1.1"
-reqwest = { version = "0.11.10", features = ["blocking"] }
+reqwest = { version = "0.11.10", features = ["rustls-tls"] }
```

Comment:
Why enable `rustls-tls`? IIRC the native TLS will actually still exist as a dependency and  it pulls in quite a few extra dependencies.

Reply:
we might need a `no-default-features = true` flag passed in then to make sure to not build the openssl backend in addition to rustls-tls.

Reply:
Ah you are correct https://github.com/rust-minidump/rust-minidump/blob/main/breakpad-symbols/Cargo.toml#L30